upi/vsphere/vm/ignition.tf

```diff
@@ -18,12 +18,12 @@ data "ignition_file" "static_ip" {
   for_each = var.hostnames_ip_addresses
 
   filesystem = "root"
-  path       = "/etc/sysconfig/network-scripts/ifcfg-ens192"
+  path       = "/etc/NetworkManager/system-connections/eth0.nmconnection"
```

Comment:
@LorbusChris like @dustymabe mentioned this needs to be `ens192` and in the template.

Reply:
I just changed the name in the last commit to test if that changes anything, it was ens192 before and didn't work either :( https://github.com/openshift/installer/pull/3533/commits/1c80ea865b8083e212028c870edf0f8abaae3196

Reply:
Was it like:
"/etc/NetworkManager/system-connections/ens192.nmconnection"
and that didn't work?

I haven't used this method of configuring adapters before so /shrug :)